### ISSUE TITLE
CLI app installation times out when you do not include a notes.txt file

### ISSUE THREAD

**Issue by JasonvanBrackel** [rancher/rancher#16201]
**What kind of request is this (question/bug/enhancement/feature request):**
feature request

**Steps to reproduce (least amount of steps as possible):**
Some applications that take a long time to install, will cause this an error to occur.

**Result:**
https://github.com/rancher/cli/search?q=timed+out+waiting+for+app+notes%2C+the+app+could+still+be+installing.+Run+%27rancher+apps%27+to+verify&unscoped_q=timed+out+waiting+for+app+notes%2C+the+app+could+still+be+installing.+Run+%27rancher+apps%27+to+verify

**Environment information**
Rancher CLI v.2.0.5

**Comment by josephmcgovern-wf** [comment]
The command we are running is:
```
rancher apps install <chart> <service> --version=<version> --answers=<path_to_answers.yaml> --namespace=<namespace>
```

We hit this timeout and are notified that the deploy fails. However, running `rancher apps ls` while the above process is waiting reveals that the app has indeed become `active`.

We do not include a `template/NOTES.txt` in the chart definition. I wonder if [this line](https://github.com/rancher/cli/blob/baf6d6074e9b5cb6dca1abef24e474a5823aa464/cmd/app.go#L590) should be changed to not require app notes at all.

**Comment by JasonvanBrackel** [comment]
From a conversation I had yesterday with @josephmcgovern-wf, the process succeeds if you include NOTES.txt.  This is optional so our routine should take that into account.

**Comment by cjellick** [comment]
merged. on monday remind me to cut an rc of the cli @erikwilson

**Comment by jiaqiluo** [comment]
The bug is reproduced on CLI v.2.0.5
Result: Although the app is active the command line is still waiting and eventually prints an error message.

The bug fix is validated on CLI v2.2.0-rc5
Result: The command line is returned as soon as the app is active.
